cf_units/_udunits2_parser/udunits2Parser.g4

```diff
@@ -1,6 +1,79 @@
 parser grammar udunits2Parser;
-  
+
 options { tokenVocab=udunits2Lexer; }
 
 
-unit_spec: ALL;
+unit_spec:
+    shift_spec? EOF  // Zero or one "shift_spec", followed by the end of the input.
+;
+
+shift_spec:
+    product
+    | product shift number    // Kelvin @ 273.15
+    | product shift timestamp // hours since 2001-12-31 23:59:59.999 +6
+;
+
+product:
+    power
+    | product power             // e.g. m2s (s*m^2)
+    | product MULTIPLY power    // e.g. m2*s
+    | product DIVIDE power      // e.g. m2/2
+    | product WS+ power         // e.g. "m2 s"
```

Comment:
@pelson So the UDUNITS-2 grammar doesn't explicitly define this last case, and yet supports it? Is that right?

Reply:
Yes. Also space is defined in the MULTIPLY token in the C grammar, but that isn't actually accurate (whitespace is significant in other parts of the codebase).